sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/__init__.py

```diff
@@ -26,8 +26,8 @@
     TextDocumentBatchStatistics,
     SentenceSentiment,
     SentimentConfidenceScores,
-    SentenceAspect,
-    AspectOpinion
+    AspectSentiment,
```

Comment:
Would it be better to call this SentimentAspect and SentimentOpinion as this represents the `Aspect of the Sentiment` and not the `Aspect's sentiment`.
I feel with `AspectSentiment` it suggests the latter?

Reply:
We have DocumentSentiment, SentenceSentiment, so if we follow the pattern. It would be AspectSentiment and OpinionSentiment.

Reply:
Yeah after talking to @annelo-msft , she brought up a really good point where these all have the same basic structure (have `confidence_scores`, `sentiment` etc), and it's more important for a user to have a logical pattern than for the individual names to be the best English.